src/NuGet.Core/NuGet.Configuration/PackageSource/NuGetConstants.cs

```diff
@@ -40,6 +40,8 @@ public static class NuGetConstants
 
         public static readonly string FeedName = "nuget.org";
 
+        public static readonly string AddV3TrackFile = "nugetorgadd.trk";
```

Comment:
I like to use nouns in properties constants. If I see a verb, it suggest me an action.

```suggestion
        public static readonly string V3TrackFile = "nugetorgadd.trk";
```

Reply:
As the previous comment said, this is what we had in 5.11 and earlier. Although honestly I feel like this is an example of something NuGet should never had made public. I don't understand why anyone using our package should care about this file. As far as I'm concerned, it's an internal NuGet concern, not useful for others using our package.